src/birthder.js

```diff
@@ -33,8 +33,8 @@
     const BIRTHDAY_ANNOUNCEMENT_BEFORE_MODE = process.env.BIRTHDAY_ANNOUNCEMENT_BEFORE_MODE || false;
 
     const MSG_PERMISSION_DENIED = "Permission denied.";
-    const DATE_FORMAT = "DD/MM/YYYY";
-    const SHORT_DATE_FORMAT = "DD/MM";
```

Comment:
I can understand why you don't like the format, but the changes is not connected directly to the problem the PR is trying to solve, so please return the previous format (i.e. `DD/MM`).

Reply:
Ok, I'll fix it.